testsuite/forge/Cargo.toml

```diff
@@ -15,7 +15,7 @@ rust-version = { workspace = true }
 [dependencies]
 again = { workspace = true }
 anyhow = { workspace = true, features = ["backtrace"] }
-aptos = { workspace = true }
```

Comment:
This is a zombie file resurrected by merge. As the crate does not exist in the movement fork, could be removed.